Solutions/CiscoSecureEndpoint/Analytic Rules/CiscoEndpointHighAlert.yaml

```diff
@@ -0,0 +1,38 @@
+id: 4683ebce-07ad-4089-89e3-39d8fe83c011
+name: Cisco SE High Events Last Hour
+description: |
+  'Find events from Cisco Secure Endpoint that are of High severity in the last hour.'
+severity: High
+requiredDataConnectors:
+  - connectorId: CiscoSecureEndpoint
+    dataTypes:
+      - CiscoSecureEndpoint_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+  - InitialAccess
+query: |
+  let endtime = 1h;
```

Comment:
Since the Query Frequency is set to 1 hr . Wondering if we need the endtime value.

Reply:
We don't if the user doesn't change the timeframe. However, if they do without thinking, it could be a really long-running query so I kept it in there to prevent this.